Userland/Libraries/LibGfx/Vector2.h

```diff
@@ -105,7 +106,10 @@ class Vector2 final {
 
     constexpr void normalize()
     {
-        T inv_length = 1 / length();
+        auto len = length();
+        if (len == 0) [[unlikely]]
```

Comment:
Can we move this check and the others for Vector{2,3,4} into the draw_primitives function where the lighting is calculated? I think it should be up to the user of the class to decide what happens in case of a zero length vector.

Reply:
Yes - all `Vector` changes were reverted.